internal/cmd/module/local_preview.go

```diff
@@ -51,7 +51,7 @@ func localPreview() cli.ActionFunc {
 
 		fp := filepath.Join(os.TempDir(), "spacectl", "local-workspace", fmt.Sprintf("%s.tar.gz", uploadMutation.UploadLocalWorkspace.ID))
 
-		matchFn, err := internal.GetIgnoreMatcherFn(ctx)
+		matchFn, err := internal.GetIgnoreMatcherFn(ctx, "")
```

Comment:
I think we should switch off this behaviour for modules. I know they are generally handled slightly differently be the run side orchestrator?